giskard/datasets/base/__init__.py

```diff
@@ -507,6 +507,19 @@ def upload(self, client: GiskardClient, project_key: str):
             )
         return dataset_id
 
+    def extract_languages(self):
+        """
+        Extracts all languages present in the dataset 'text' column.
+
+        Returns:
+            list[str]: a list of language codescontaining all languages in the dataset.
+        """
+        languages_set = []
```

Comment:
Naming? `language_set` is a `list`. This may be a bit confusing. You can actually use directly a `set`.
Also, no need to include the type in the name of the variable, the code should be explicit enough:
```py3
languages = set()
```